### DISCUSSION TITLE
How do you remove a 3rd party repo?

### DISCUSSION THREAD

**Discussion by trymeouteh** [pacstall/pacstall#550] [Q&A] [answered]
How do you remove a 3rd party repo? I do not see any commands to achieve this.

**Comment by wizard-28** [comment, marked as answer]
Yeah we do not have a command to remove it at the moment, but fortunately it's easy enough to do.

1. Edit `/usr/share/pacstall/repo/pacstallrepo.txt`
2. Delete the link of the 3rd party repository that you wish to remove. (Don't remove the `https://raw.githubusercontent.com/pacstall/pacstall-programs/master` one as it's the official one)
